WordPress/gutenberg
# Add author name block

## Description

Split the author name into its own block.
Partial for https://github.com/WordPress/gutenberg/issues/35596

A decision is needed about the most appropriate HTML tag for displaying the name.

## How has this been tested?




1. Add the new block in the editor.
2. Confirm that the authors display name is showing.
3. Try the alignment settings.
4. Try the link setting.


## Types of changes




New block

## Checklist:
